src/ui/macstr_ui.py

```diff
@@ -1,214 +1,159 @@
-        self.radioButton_6.setGeometry(QtCore.QRect(530, 210, 106, 23))
-        self.radioButton_6.setObjectName("radioButton_6")
-
-        self.radioButton_7 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_7.setGeometry(QtCore.QRect(140, 460, 106, 23))
-        self.radioButton_7.setObjectName("radioButton_7")
+        self.refresh_btn = QtWidgets.QPushButton(self.centralwidget)
+        self.refresh_btn.setGeometry(QtCore.QRect(350, 510, 83, 25))
+        self.refresh_btn.setObjectName("refresh_btn")
 
-        self.radioButton_8 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_8.setGeometry(QtCore.QRect(30, 460, 106, 23))
-        self.radioButton_8.setObjectName("radioButton_8")
-
-        self.radioButton_9 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_9.setGeometry(QtCore.QRect(390, 460, 106, 23))
-        self.radioButton_9.setObjectName("radioButton_9")
-
-        self.radioButton_10 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_10.setGeometry(QtCore.QRect(280, 460, 106, 23))
-        self.radioButton_10.setObjectName("radioButton_10")
-
-        self.radioButton_11 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_11.setGeometry(QtCore.QRect(640, 460, 106, 23))
-        self.radioButton_11.setObjectName("radioButton_11")
-
-        self.radioButton_12 = QtWidgets.QRadioButton(self.centralwidget)
-        self.radioButton_12.setGeometry(QtCore.QRect(530, 460, 106, 23))
-        self.radioButton_12.setObjectName("radioButton_12")
+        # ------------------------------
+        # Creating toggle buttons
+        # ------------------------------
+        self.btn1 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn1.setGeometry(QtCore.QRect(90, 210, 83, 25))
+        self.btn1.setDefault(False)
+        self.btn1.setObjectName("btn1")
+        self.btn2 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn2.setGeometry(QtCore.QRect(350, 210, 83, 25))
+        self.btn2.setProperty("button_id", 2)
+        self.btn2.setObjectName("btn2")
+        self.btn3 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn3.setGeometry(QtCore.QRect(590, 210, 83, 25))
+        self.btn3.setProperty("button_id", 3)
+        self.btn3.setObjectName("btn3")
+        self.btn4 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn4.setGeometry(QtCore.QRect(90, 460, 83, 25))
+        self.btn4.setProperty("button_id", 4)
+        self.btn4.setObjectName("btn4")
+        self.btn5 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn5.setGeometry(QtCore.QRect(350, 460, 83, 25))
+        self.btn5.setProperty("button_id", 5)
+        self.btn5.setObjectName("btn5")
+        self.btn6 = QtWidgets.QPushButton(self.centralwidget)
+        self.btn6.setGeometry(QtCore.QRect(590, 460, 83, 25))
+        self.btn6.setProperty("button_id", 6)
+        self.btn6.setObjectName("btn6")
```

Comment:
Good suggestion! However, were you referring to set in within the  the `.setProperty` or `setObjectName` method call or both?

Reply:
@axiomcura - apologies, I mostly mean the base Python attribute name (and not the more specific Qt properties). The property or object names might also use more specificity, and perhaps this could stem from the attribute name.

For ex:
```python
self.btn1 = QtWidgets.QPushButton(self.centralwidget)
```

Could become something like:
```python
self. toggle_btn1 = QtWidgets.QPushButton(self.centralwidget)
```

It might also be useful to consider using a naming scheme which builds on a common base from most broad to most specific. Adding to the specificity could mean relative location or purpose (instead of a number), but this could be unreasonable so please use your best judgement.

```python
self. btn_toggle_topleft = QtWidgets.QPushButton(self.centralwidget)
```